fluentd/configs.d/filter/k8s_record_transform.conf

```diff
@@ -1,4 +1,24 @@
-<filter kubernetes.**>
+<filter kubernetes.journal.container**>
+  type record_transformer
+  enable_ruby
+  <record>
+    hostname ${(kubernetes_host rescue nil) || File.open('/etc/docker-hostname') { |f| f.readline }.rstrip}
+    # log is set if MESSAGE is JSON valued and has err or msg field
+    message ${record['MESSAGE'] || log}
+    version 1.2.0
```

Comment:
@ewolinetz This commit does not introduce any formatting changes AFAIK.  I wanted to make the output, when using journal, identical to the output when reading from files, for ease of migration.  The only change is that the timestamp now has microsecond resolution when reading from journal and writing to the `.operations.*` index.